.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+### Instructions
+
+- Determine whether your issue is a **bug report**, a **feature request**, or a **documentation request** (for tool gatkdocs only; for forum docs see [the support issues repo](https://github.com/broadinstitute/dsde-docs/issues));
+- Consider if your "issue" is better addressed on the GATK forum: http://gatkforums.broadinstitute.org/gatk  Post there if you have questions about expected tool behavior, output format, unexpected results or generally any question that does not fit the categories above;
+- Use a **concise** yet **descriptive** title;
```

Comment:
I would probably add a line suggesting to first search the issues to see if there's an existing ticket.